solitude/urls.py

```diff
@@ -37,13 +19,15 @@
 )
 
 urls = [
+    url(r'^$', 'solitude.views.home', name='home'),
+    url(r'^generic/', include('lib.buyers.urls', namespace='buyer')),
```

Comment:
added in https://github.com/andymckay/solitude/commit/4c8768a23947bd1c083a70d5070d3462d4f5de20